test/molecule-role/molecule/vms/tests/test_receiver_topology.py

```diff
@@ -108,7 +108,7 @@ def wait_for_components():
         with open("./topic-topo-process-agents-dnat.json", 'w') as f:
             json.dump(json_data, f, indent=4)
 
-        endpoint_match = re.compile("urn:endpoint:/.*:{}".format(ubuntu_private_ip))
+        endpoint_match = re.compile("urn:endpoint:/{}".format(ubuntu_private_ip))
```

Comment:
why has this changed?

Reply:
due to the fact that we do not produce a namespace when the connection is not on loopback. this was basically triggering the bug that i fixed